packages/manager/modules/hub/src/components/products/products.component.js

```diff
@@ -3,6 +3,7 @@ import template from './products.html';
 export default {
   bindings: {
     products: '<',
+    seeAll: '&',
```

Comment:
Just a side note, i think something like 'onProductSelect' is better because the component should not have the information on the action that will occur when a product is selected. Knowing the "seeAll" action is already on a higher level (unless it's labelled "see all' in the component, in that case it's valid). Just being picky, not mandatory

Reply:
@frenautvh I agree with you it makes sense to change to something less parent specific. I'll update as the change is quite easy to make :)